PhysicsTools/SelectorUtils/interface/PFJetIDSelectionFunctor.h

```diff
@@ -243,40 +282,51 @@ class PFJetIDSelectionFunctor : public Selector<pat::Jet>  {
 
     push_back("CHF" );
     push_back("NHF" );
```

Comment:
for a future PR I made #23131 to hopefully reduce the duplicated implementation of the cuts